src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -2514,7 +2523,7 @@ protected boolean canGenerateReport( List<String> files )
      * Method to get the excluded source files from the javadoc and create the argument string
      * that will be included in the javadoc commandline execution.
      *
-     * @param sourcePaths the collection of paths to the source files
+     * @param sourceFolders the collection of paths to the source files

```

Comment:
Minor bugfix: The Javadoc parameter name did not match the method parameter name.